nautobot_circuit_maintenance/handle_notifications/handler.py

```diff
@@ -185,9 +185,15 @@ def process_raw_notification(  # pylint: disable=too-many-branches
         )
         return None
 
+    provider_type = (
+        provider.get_custom_fields().get(CustomField.objects.get(name="provider_parser_circuit_maintenances"))
```

Comment:
I *think* this could be implemented more simply as `provider.cf.get("provider_parser_circuit_maintenances")`?